world-server/src/network/Server.ts

```diff
@@ -50,10 +54,13 @@ export class Server implements Messages.MsgHandler {
         var connection = request.accept('', request.origin);
         Logger.info(`Accepted socket connection from ${request.remoteAddress}`)
 
-        connection.internalId = this.connCounter++;
+        // Commented out for now to avoid build errors
+        // the library should do this automatically if debug is enabled
+        // https://github.com/theturtle32/WebSocket-Node/blob/1f7ffba2f7a6f9473bcb39228264380ce2772ba7/lib/WebSocketConnection.js#L42
+        // connection.internalId = this.connCounter++;
```

Comment:
We will need to track our internal ID on the connection itself so we must find a solution for this problem. I don't want to wrap this into our own class. Can we do something like
```typescript
connection['internalId'] = this.connCounter++;
```

Reply:
I looked at a couple of things but TypeScript disallows this kind of dynamic property adding. The only solution I see, if we really want to use the `WebSocketConnection` class, is that we fork the repo and add our own `internalId` to the class.